frontend/root_words_ui/components/Form.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {InputLabel, Button, Box} from '@mui/material/';
+import MenuItem from '@mui/material/MenuItem';
+import FormHelperText from '@mui/material/FormHelperText';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+
+export default function Form() {
+  const [chapter, setChapter] = React.useState('');
+  const [verse, setVerse] = React.useState('');
+
+  const handleChapterChange = (event) => {
+    setChapter(event.target.value);
+  };
+
+  const handleVerseChange = (event) => {
+    setVerse(event.target.value)
+  };
+
+    return (
+    <div>
+      <Box display="flex" justifyContent="center">  
+      <FormControl required sx={{ m: 1, minWidth: 120 }}>
+        <InputLabel id="demo-simple-select-required-label">Chapter</InputLabel>
+        <Select
+          labelId="demo-simple-select-required-label"
+          id="demo-simple-select-required"
+          value={chapter}
+          label="Chapter"
+          onChange={handleChapterChange}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
```

Comment:
Consider how we render the list of items inside of chapterinput.js. Remember that when we make an API request, we will receive all of the chapter names/numbers (i.e. `1. Fatiha`). The same goes for the verses down below.